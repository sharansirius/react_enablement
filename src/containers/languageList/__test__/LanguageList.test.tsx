import React from "react";
import renderer from "react-test-renderer";
import LanguageList from "../languageList";

it("matches snapshot", () => {
  const tree = renderer.create(<LanguageList />).toJSON();
  expect(tree).toMatchSnapshot();
});
