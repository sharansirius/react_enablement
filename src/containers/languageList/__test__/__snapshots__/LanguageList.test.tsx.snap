// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="languageList"
>
  <h2
    className="heading small"
  >
    View in Other Languages
  </h2>
  <div
    className="content"
  >
    <div
      className="language"
    >
      E
    </div>
    <div
      className="language"
    >
      த
    </div>
    <div
      className="language"
    >
      త
    </div>
    <div
      className="language"
    >
      മ
    </div>
    <div
      className="language"
    >
      ह
    </div>
  </div>
</div>
`;
